### PR TITLE
add check to upgrade to warn for not supported php version; disable geoip for php <5.4

### PR DESCRIPTION
**Pull request by Skywalker-11**
#1073

### CODE REVIEW

File: mailscanner/languages/fr.php [hunk 1]
```diff
@@ -267,6 +267,7 @@
     'report04' => 'Report :',
     'spamassassin04' => 'SpamAssassin',
     'spamassassinmcp04' => 'SpamAssassin MCP',
+    'geoipnotsupported04' => 'GeoIP not supported',
```
Comment (stefaweb) [inline mailscanner/languages/fr.php]: 'geoipnotsupported04' => 'GeoIP non pris en charge',

File: upgrade.php [hunk 2]
```diff
@@ -998,6 +998,15 @@ function stringEndsWith($haystack, $needles)
     $errors[] = $err_msg;
 }
 
+echo PHP_EOL;
+//Check minimal PHP version
+echo pad(' - Checking minimal PHP version >= 5.4');
+if (version_compare(phpversion(), '5.4.0', '>=')) {
+    echo color(' PHP version OK', 'lightgreen') . PHP_EOL;
+} else {
+    echo color(' WARNING: PHP version < 5.4 not fully supported (GeoLite2 not working)', 'red'). PHP_EOL;
```
Comment (endelwar) [inline upgrade.php]: I would issue a yellow warning instead of a red error for this

**Review by endelwar — CHANGES_REQUESTED** [review]

**Review by endelwar — APPROVED** [review]

**Comment by Skywalker-11** [comment]
@endelwar can you also take a look at the PRs of the doc?

**Comment by endelwar** [comment]
https://github.com/mailwatch/mailwatch-docs/pull/36 merged

**Pull request merged**